<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEN-Builder</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PlacementBoard" name="placementBoard">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>325</width>
          <height>355</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>325</width>
          <height>355</height>
         </size>
        </property>
        <property name="title">
         <string>Board</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="SelectionBox" name="selectionBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>325</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>325</height>
             </size>
            </property>
            <property name="title">
             <string>Pieces</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="nextMoveGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Next to Move</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QRadioButton" name="wRadioButton">
               <property name="text">
                <string>white</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="bRadioButton">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>black</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="castlingGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>castling options</string>
            </property>
            <property name="title">
             <string>Castling</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="castlingLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QCheckBox" name="wkCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>O-O</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wqCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>O-O-O</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="bkCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>O-O</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="bqCheckBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>O-O-O</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="enPassentGroupBox">
            <property name="toolTip">
             <string>options to make an en passant capture</string>
            </property>
            <property name="title">
             <string>En Passant</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QListWidget" name="enPassantListWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="moveGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>full moves from start of game</string>
            </property>
            <property name="title">
             <string>Move</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QSpinBox" name="moveSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>16777215</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="clockGroupBox">
            <property name="toolTip">
             <string>halfmoves since the last capture or pawn advance</string>
            </property>
            <property name="title">
             <string>Clock</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QSpinBox" name="clockSpinBox">
               <property name="maximum">
                <number>16777215</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFEN">
    <property name="title">
     <string>FEN</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Board</string>
    </property>
    <addaction name="actionResetBoard"/>
    <addaction name="actionClearBoard"/>
    <addaction name="separator"/>
    <addaction name="actionFlipBoard"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuFEN"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionResetBoard">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionBlack_King">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black King</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionWhite_King">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White King</string>
   </property>
   <property name="shortcut">
    <string>Shift+K</string>
   </property>
  </action>
  <action name="actionWhite_Queen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White Queen</string>
   </property>
   <property name="shortcut">
    <string>Shift+Q</string>
   </property>
  </action>
  <action name="actionBlack_Queen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black Queen</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionWhite_Rook">
   <property name="text">
    <string>White Rook</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionBlack_Rook">
   <property name="text">
    <string>Black Rook</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionWhite_Bishop">
   <property name="text">
    <string>White Bishop</string>
   </property>
   <property name="shortcut">
    <string>Shift+B</string>
   </property>
  </action>
  <action name="actionBlack_Bishop">
   <property name="text">
    <string>Black Bishop</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionWhite_Knight">
   <property name="text">
    <string>White Knight</string>
   </property>
   <property name="shortcut">
    <string>Shift+N</string>
   </property>
  </action>
  <action name="actionBlack_Knight">
   <property name="text">
    <string>Black Knight</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionWhite_Pawn">
   <property name="text">
    <string>White Pawn</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionBlack_Pawn">
   <property name="text">
    <string>Black Pawn</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionClearBoard">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionFlipBoard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flipped</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlacementBoard</class>
   <extends>QGroupBox</extends>
   <header location="global">qbuildfen</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectionBox</class>
   <extends>QGroupBox</extends>
   <header location="global">qbuildfen</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
